cmake/O2Dependencies.cmake

```diff
@@ -406,6 +426,10 @@ o2_define_bucket(
     ${ROOT_INCLUDE_DIR}
 )
 
+if(FairLogger_FOUND)
+  set(FairLogger_DEP FairLogger::FairLogger)
+endif()
```

Comment:
I would remove the `if` clause here because FairLogger is a required dependency anyhow, and this makes the definition of `FairLogger_DEP` obsolete. Simply keep `FairLogger::FairLogger` as dependency.

Reply:
It is conditional, because the idea was to still support building against older FairRoot versions before the split.